config/locales/views/admin/en.yml

```diff
@@ -113,6 +113,7 @@ en:
             select: Select role
             note: "Add a note to this action:"
             submit: Add
+            warning: "Comma Delimited, All Lowercase, No @ Symbol Required In Front Of Usernames Warning: Usernames are case-sensitive and must be all lowercase! This task will fail silently when supplied with incorrect usernames."
```

Comment:
This messaging makes me wonder if the _whole task_ fails when I get one name wrong, or if it's only _part_ of the task that fails?

Reply:
This is actually the warning which show above the `Username` box. https://user-images.githubusercontent.com/9396084/252765869-a3c6b2c0-4657-4660-9c09-1ec97b1ea7f5.png

So its actually not related to API request.